Installing Guest Attestation extension by default for Trusted Launch enabled Linux VMs

sirfamelin opened the pull request:
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)


- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Adding changes to install the Guest Attestation extension for Trusted Launch Linux VMs with Secureboot and vTPM enabled.

azure-quickstarts commented:
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

bmoore-msft reviewed (changes requested):
Let's update the readme with some detail about these changes - also check the pipeline results for other changes that need to be made.

azure-quickstarts commented:
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

azure-quickstarts commented:
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

azure-quickstarts commented:
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

sirfamelin commented:
> Let's update the readme with some detail about these changes - also check the pipeline results for other changes that need to be made.

Updated the readme and addressed all the pipeline checks

azure-quickstarts commented:
@cynthn - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]
